docs/drivers.rst

```diff
@@ -81,6 +81,7 @@ Helpers for building graphical interfaces using the displayio core module and fr
     Turtle Graphics <https://circuitpython.readthedocs.io/projects/turtle/en/latest/>
     WSGI <https://circuitpython.readthedocs.io/projects/wsgi/en/latest/>
     DisplayIO Layout <https://circuitpython.readthedocs.io/projects/displayio-layout/en/latest/>
+    DisplayIO FlipClock <https://readthedocs.org/projects/adafruit-circuitpython-displayio-flipclock/>
```

Comment:
Looks like you added the RTD project page instead of the docs themelves

Reply:
I was unable to find a URL with the same format as the others that worked. This one is what it seems like it would be: `https://circuitpython.readthedocs.io/projects/displayio-flipclock/en/latest/` but it doesn't find it. 

I did find this one, which is a bit different with subdomains but does lead to the docs: `https://adafruit-circuitpython-displayio-flipclock.readthedocs.io/en/latest/` will change it to that.

Reply:
Never mind I'm not sure I can, but since you have access to the library, you can:  https://readthedocs.org/dashboard/circuitpython/subprojects/create/.  Webhook is still needed though.

Reply:
I don't think I've done the subproject addition process before. Do you know if that is documented somewhere?

Reply:
I don't think so, just instruction by word-of-mouth from @kattni.  It might be a good idea to add it to the Learn Guide, but I also know that non-Adafruit people use it so it might be confusing for them.

Reply:
Please do not change the link. That it is not working is likely on me. There's a huge part of it that anyone can do, and @FoamyGuy I'll walk you through that. I need to check your permissions, you may be able to do the whole process. If not, the last step will be on me.

@FoamyGuy If you're available after the CP Weekly meeting today, we can chat to go through this, if you're interested.

Reply:
@kattni okay sounds good thank you. I'll be around and am happy to learn how.